components/raftstore/src/store/msg.rs

```diff
@@ -308,6 +308,25 @@ where
         context: GetEntriesContext,
         res: Box<RaftlogFetchResult>,
     },
+    EnterForceLeaderState,
+    ExitForceLeaderState,
+}
+
+#[derive(Debug)]
+pub enum RaftlogFetchResult {
```

Comment:
question: where is it used? besides fetching and fetched, is it possible to fail?

Reply:
unrelated with this PR, see https://github.com/tikv/tikv/pull/11409
And it's possible to fail returning with ents of `raft::Result`